### ISSUE TITLE
 loss suddenly turned into nan

### ISSUE THREAD

**Issue by Z-zhe** [JiahuiYu/wdsr_ntire2018#36]
Hi, when I train with lr=1e-4, batch_size=16, out_patch_size=192, my loss was normal, after about 15,000 steps (1 epoch = 30,000 steps), the loss suddenly turned into nan, my data was clean, and I used L1Loss. How should this problem be solved? thanks

**Comment by JiahuiYu** [comment]
@Z-zhe In that case for your dataset, I would suggest lower learning rate, maybe by half. Another option is to use warm start learning rates.
